submission/submitter/src/main/kotlin/ac/uk/ebi/biostd/submission/submitter/SubmissionSubmitter.kt

```diff
@@ -22,14 +22,14 @@ class SubmissionSubmitter(
 ) {
     fun submit(rqt: SubmitRequest): ExtSubmission {
         val submission = processRequest(rqt)
-        val (accNo, version) = submissionSubmitter.createRequest(ExtSubmitRequest(submission, rqt.draftKey))
+        val (accNo, version) = submissionSubmitter.createRequest(ExtSubmitRequest(submission, rqt.owner, rqt.draftKey))
         submissionSubmitter.handleRequest(accNo, version)
         return submission
     }
 
     fun createRequest(rqt: SubmitRequest): ExtSubmission {
         val submission = processRequest(rqt)
-        submissionSubmitter.createRequest(ExtSubmitRequest(submission, rqt.draftKey))
+        submissionSubmitter.createRequest(ExtSubmitRequest(submission, rqt.owner, rqt.draftKey))
```

Comment:
I found really confusing that at request level we are defining owner as:

` val owner: String = onBehalfUser?.email ?: submitter.email`

but when processing submission it is defined as.

` val ownerEmail = onBehalfUser?.email ?: previousVersion?.owner ?: submitter.email`

It looks to me that the first one is not the owner but rather the `notifiedUser`